### ISSUE TITLE
remove reduction folder

### ISSUE THREAD

**Issue by mantid-builder** [mantidproject/mantid#9656]
sans_code
This issue was originally [TRAC 8812](http://trac.mantidproject.org/mantid/ticket/8812)

Original Reporter: Gesner Passos

This ticket is blocked by : 
- [X] [TRAC9014](http://trac.mantidproject.org/mantid/ticket/9014)
- [ ] [TRAC9091](http://trac.mantidproject.org/mantid/ticket/9091)
- [X] [TRAC9093](http://trac.mantidproject.org/mantid/ticket/9093)

This ticket is blocks :  [TRAC4259](http://trac.mantidproject.org/mantid/ticket/4259)

It is necessary to move what we use to SANS folder and then remove the reduction folder: scripts/reduction

**Comment by AntonPiccardoSelg** [comment]
not relevant for the new reduction backend. The folder will be removed once the old reduction system gets removed. Does not need a separate ticket
